src/components/chat/UserStatusIcons.js

```diff
@@ -11,9 +11,9 @@ export const ViewingUserIcon = (props) => {
     <div style={styleUserStatus}>
       <FaCircle style={styleViewing} />
       {username}
-      {isCurrentUser && <span>&nbsp;(me)</span> }
+      {isCurrentUser && <span>{' '}(me)</span> }
```

Comment:
@jimmcgaw For some reason this change doesn't behave the same way. I see no gap between my username and `(me)`, do you?

Reply:
Ah, from an earlier iteration. I can revert that here, but will be fixed in https://github.com/cryptag/leapchat/pull/277/files if we merge that fix and fix conflicts.

Reply:
Sounds good

Reply:
Looks like merging will pull this in. Change it here, por favor?